aioseop_class.php

```diff
@@ -3350,8 +3350,9 @@ public function admin_enqueue_scripts( $hook_suffix ) {
 		// This ensures different JS files are enqueued only at the intended screens. Preventing unnecessary processes.
 		$extra_title_len = 0;
 		switch ( $hook_suffix ) {
-			// Screens `post.php` & `./aioseop_class.php` share the same `count-char.js`.
+			// Screens `post.php`, `post-new.php`, & `../aioseop_class.php` share the same `count-char.js`.
 			case 'post.php' :
+			case 'post-new.php' :
```

Comment:
add `// fall through` here as well

Reply:
I don't think it does on this one, are you absolutely sure? https://eslint.org/docs/rules/no-fallthrough

Reply:
@EkoJR grunt might not dislike it, I'm not sure, but I find it helpful to state a fall through